www/webapp/src/router/index.js

```diff
@@ -15,24 +15,24 @@ const routes = [
     // route level code-splitting
     // this generates a separate chunk (about.[hash].js) for this route
     // which is lazy-loaded when the route is visited.
-    component: () => import(/* webpackChunkName: "signup" */ '@/views/SignUp.vue')
```

Comment:
`vue-cli` (webpack) created a big js. The comment `webpackChunkName` it was possible to split this big js into smaller chunks.
`vite` (rollup) transforms every source code into a chunk. So it creates automatically the smallest possible chunk. If required you can summarize multiple small chunks into a bigger to avoid a lot of requests to the server or delays by requests.

Example build output (partial):

![grafik](https://github.com/desec-io/desec-stack/assets/7337347/2839a655-0d89-4451-b27a-9ff77c60e14f)

Reply:
Aha. I think that's fine for now. Earlier, you put quite some effort into reducing latency -- I trust that in case the many requests pose a problem, you're probably going to do something about it :p